benches/bench.rs

```diff
@@ -401,6 +401,7 @@ fn write_b_bench(b: &mut Bencher, tx_size: TxSize, qindex: usize) {
         x: sbx,
         y: sby
       };
+      fs.qc.update(fi.qindex, tx_size);
```

Comment:
This could have been immutable instead, e.g. part of the FrameInvariants constructor.